internal/pgbouncer/reconcile.go

```diff
@@ -126,60 +126,35 @@ func Pod(
 		return
 	}
 
-	backend := corev1.Volume{Name: "pgbouncer-backend-tls"}
-	backend.Projected = &corev1.ProjectedVolumeSource{
-		Sources: []corev1.VolumeProjection{
-			backendAuthority(inPostgreSQLCertificate),
-		},
-	}
-
-	frontend := corev1.Volume{Name: "pgbouncer-frontend-tls"}
-	frontend.Projected = &corev1.ProjectedVolumeSource{
-		Sources: []corev1.VolumeProjection{
-			frontendCertificate(
-				inCluster.Spec.Proxy.PGBouncer.CustomTLSSecret, inSecret),
-		},
+	configVolumeMount := corev1.VolumeMount{
+		Name: "pgbouncer-config", MountPath: configDirectory, ReadOnly: true,
 	}
-
-	configVol := corev1.Volume{Name: "pgbouncer-config"}
-	configVol.Projected = &corev1.ProjectedVolumeSource{
-		Sources: podConfigFiles(
-			inCluster.Spec.Proxy.PGBouncer.Config, inConfigMap, inSecret),
+	configVolume := corev1.Volume{Name: configVolumeMount.Name}
+	configVolume.Projected = &corev1.ProjectedVolumeSource{
+		Sources: append(append([]corev1.VolumeProjection{},
+			podConfigFiles(inCluster.Spec.Proxy.PGBouncer.Config, inConfigMap, inSecret)...),
+			frontendCertificate(inCluster.Spec.Proxy.PGBouncer.CustomTLSSecret, inSecret),
+			backendAuthority(inPostgreSQLCertificate),
+		),
 	}
 
 	container := corev1.Container{
 		Name: naming.ContainerPGBouncer,
 
-		Command:         []string{"pgbouncer", iniFileAbsolutePath},
+		Command: []string{"pgbouncer", iniFileAbsolutePath},
+
```

Comment:
⛏️ why the blank line?

Reply:
Hmm. I guess I was trying to make the two containers consistent. It looks like noise in the diff now.